statsmodels/survey/summary_stats.py

```diff
@@ -0,0 +1,612 @@
+            of data.
+        vcov : ndarray
+            The variance-covariance of the estimates.
+        """
+        est = self._stat(self.design.weights)
+
+        jdata = []
+        self.w_li = []
+        for i in range(replicates):
+            w = self.design.get_rep_weights(i, bsn=bsn)
+            self.w_li.append(w)
+            jdata.append(self._stat(w))
+        jdata = np.asarray(jdata)
+        if self.mse:
+            print("mse specified")
+            var = np.dot((jdata-est).T, (jdata-est)) / replicates
+            var = np.sqrt(np.diag(var))
+        else:
+            boot_mean = jdata.mean(0)
+            var = ((jdata - boot_mean)**2).sum(0) / replicates
+
+        return est, np.sqrt(var)
+
+    def _jackknife(self):
+        """
+        Jackknife variance estimation for survey data.
+
+        Parameters
+        ----------
+        stat : object
+            Object of class SurveyMean, SurveyTotal, SurveyPercentile, etc
+
+        Returns
+        -------
+        est : ndarray
+            The point estimates of the statistic, calculated on the columns
+            of data.
+        vcov : square ndarray
+            The variance-covariance matrix of the estimates, obtained using
+            the (drop 1) jackknife procedure.
+        pseudo : ndarray
+            The jackknife pseudo-values.
+        """
+        est = self._stat(self.design.weights)
+
+        jdata = []
+        try:
+            k = self.design.nclust
+        except AttributeError:
+            k = self.design.rep_weights.shape[1]
+        # for each cluster
+        for c in range(k):
+            # get jackknife weights
+            w = self.design.get_rep_weights(c=c)
+            jdata.append(self._stat(w))
+        jdata = np.asarray(jdata)
+        if self.mse:
+            print('mse specified')
+            jdata -= est
+        else:
+            if self.design.rep_weights is None:
+                for s in range(self.design.nstrat):
+                    # center the 'delete 1' statistic
+                    jdata[self.design.ii[s], :] -= jdata[self.design.ii[s],
+                                                         :].mean(0)
+            else:
+                jdata -= jdata.mean(0)
+
+        if self.design.rep_weights is None:
+            print("extra factors")
+            nh = self.design.ncs[self.design.sfclust].astype(np.float64)
+            _pseudo = jdata + nh[:, None] * (np.dot(self.design.weights,
+                                                self.data) - jdata)
+            mh = np.sqrt((nh - 1) / nh)
+            fh = np.sqrt(1 - self.design.fpc)
+            jdata = fh[:, None] * mh[:, None] * jdata
+        else:
+            nh = self.design.rep_weights.shape[1]
+            print(nh)
+            mh = np.sqrt((nh - 1) / nh)
+            jdata *= mh
+        vcov = np.dot(jdata.T, jdata)
+
+        return est, np.sqrt(np.diag(vcov))
```

Comment:
Can return est, vcov, se, or just est, vcov and let the receiver get se from vcov.